CMSgov/mint-app
# [NOREF] Split docker compose files

# NOREF

## Changes and Description

- This PR aims to clean up the docker compose files so that we can more easily spin up ONLY the services we need, and prevents having a bunch of compose files that re-define the same services in lots of different contexts
- Removes most (not all) duplicative environment variables that were in both .envrc and docker-compose files

## How to test this change

- Ensure the app works locally with `scripts/dev up`
- Ensure only the backend works with `scripts/dev up:backend`
- Ensure E2E tests pass in CI
- Ensure all server tests pass with `scripts/dev test:go`

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
